src/test/java/com/amazon/opendistroforelasticsearch/security/test/helper/rest/RestHelper.java

```diff
@@ -253,6 +254,12 @@ protected final CloseableHttpClient getHTTPClient() throws Exception {
 			hcb.setSSLSocketFactory(sslsf);
 		}
 
+		RequestConfig.Builder requestBuilder = RequestConfig.custom();
+		requestBuilder.setConnectTimeout(10000);
```

Comment:
Why is this needed?

Reply:
Reverted changes to install_demo_configuration.sh, HttpIntegrationTests.java and RestHelper.java